### ISSUE TITLE
Wrong selection of layers in ANN

### ISSUE THREAD

**Issue by aliaryan** [Palashio/libra#240]
I am using libra on google colab. I used neural network for a classification task. But libra is selecting wrong number of layer that is 5 on which I am having test accuracy of 0.63 and not selecting the layer that is 7 on which the test accuracy is 0.926. You can notice it in the screenshot below.
![Screenshot 2020-07-17 at 10 15 12 AM](https://user-images.githubusercontent.com/31564091/87749673-ea106480-c816-11ea-8ed1-a97db24a95dc.png)

**Comment by Palashio** [comment]
taking a look at this, if you compare the display (output) vs the best accuracy that's being displayed it seems like there's a discrepancy. The training/testing accuracy in the log for 5 layers doesn't seem to match up with the best layers found. 

Took a look at the logger, seems like the final output (best number of layers) is correct, and the logger is wrong. If you want to re-access the accuracies you can just do `client_name.accuracy` or` client_name.models['classification_ANN']['accuracy']`

**Comment by aliaryan** [comment]
Thank you for the clarification.

**Comment by Palashio** [comment]
So, if you do `a_client.model()['model']` you can actually access the trained model. You can then perform normal keras operations on it.

The way this works is that `a_client.model()` returns the dictionary that stores all the information that was generated in your query and then `['model']` gets the model for that query.

**Comment by aliaryan** [comment]
I did use client_name.models['classification_ANN'] to check the accuracy but I found out that it was the accuracy of incorrect selection of the model .i.e. 5 layered neural net model. And later I also used model summary to see how many neural layers did this model has chosen and it came out to be as 5. Although everything is working fine but It should have chosen the 7 layered neural net model. So is it actually selecting the right model or it has incorrect log .i.e. 7 layered model that can give the test accuracy of above 90%. I have attached the screenshots for reference.
![Screenshot 2020-07-17 at 12 42 25 PM](https://user-images.githubusercontent.com/31564091/87759299-ba1f8c00-c82b-11ea-96ed-c669b438baa5.png)
![Screenshot 2020-07-17 at 12 42 32 PM](https://user-images.githubusercontent.com/31564091/87759335-c73c7b00-c82b-11ea-908b-7ce66c9c10e8.png)
![Screenshot 2020-07-17 at 12 42 48 PM](https://user-images.githubusercontent.com/31564091/87759369-d4596a00-c82b-11ea-9789-b02794b7c63c.png)
![Screenshot 2020-07-17 at 12 43 14 PM](https://user-images.githubusercontent.com/31564091/87759374-d91e1e00-c82b-11ea-9814-d20b708a7231.png)
